include/swift/AST/Decl.h

```diff
@@ -4468,6 +4469,9 @@ class VarDecl : public AbstractStorageDecl {
     VarDeclBits.Specifier = static_cast<unsigned>(Spec);
   }
   
+  bool isInOut() const { return getSpecifier() == Specifier::InOut; }
```

Comment:
For two reasons: 

1) Multiple places are generalized to work over both VarDecls and ParamDecls and separating them would be time consuming.
2) The ownership manifesto makes affordances for `inout` as an ownership convention for lvalues (not that it's gonna happen that way eventually).